crates/dbsp/examples/json.rs

```diff
@@ -0,0 +1,69 @@
+use anyhow::Result;
+use dbsp::{circuit::CircuitConfig, operator::FilterMap, utils::Tup2, IndexedZSet, Runtime};
+use itertools::Itertools;
+use serde_json::json;
+
+type Json = ijson::IValue;
+type Weight = i64;
+
+fn parse_json(val: String) -> Json {
+    serde_json::from_str(&val).expect("failed to deserialize json")
+}
+
+fn json_field(val: Json, field: &str) -> Json {
```

Comment:
I guess our actual JSON API will return NULL rather than panic (as I assume this will do) if the field is missing.

Reply:
Yeah, we should return NULL if the field doesn't exist in the JSON.

Reply:
we should have two functions `json_field`, which panics, and `try_json_field`, which returns an Option.